stdlib/2and3/xml/dom/__init__.pyi

```diff
@@ -0,0 +1,67 @@
+class Node:
+    ELEMENT_NODE: int
+    ATTRIBUTE_NODE: int
+    TEXT_NODE: int
+    CDATA_SECTION_NODE: int
+    ENTITY_REFERENCE_NODE: int
+    ENTITY_NODE: int
+    PROCESSING_INSTRUCTION_NODE: int
+    COMMENT_NODE: int
+    DOCUMENT_NODE: int
+    DOCUMENT_TYPE_NODE: int
+    DOCUMENT_FRAGMENT_NODE: int
+    NOTATION_NODE: int
+
+
+#ExceptionCode
```

Comment:
Flake8 complains about the lack of a space behind the hash mark, the this is good to be merged.